scripts/prompt-config.sh

```diff
@@ -886,7 +887,8 @@ export TERM=xterm-color
 
 trap sig_doshell SIGINT
 
-printheader "Copyright 2025 MNX Cloud, Inc."
+BANNER_YEAR=$(TZ=UTC date "+%Y")
```

Comment:
While I too hate having to update timestamps in too many places...
If someone in the far future runs this particular version of the code, if feels like this would try to imply a copyright that may not be real. Could it perhaps grep the copyright line from a file in `/etc`?
Otherwise I think it should be hard coded. But maybe I'm overthinking this.